### ISSUE TITLE
Receiving of midi input is shown as slider in javascript & vst output

### ISSUE THREAD

**Issue by woldhek** [enzienaudio/heavy#162]
create a midi input reciever .
Compile it and notice that this midi reciever also gets a float slider in both the vst binary and webpreview.

**Comment by woldhek** [comment]
this doesnt seem to be a 100% reproducable.

**Comment by woldhek** [comment]
ah it seems that i've found it. IN the documentation there is an example patch. that shows a need to add the @hv_param markup. But in the other example it is simply just [r __hv_notein].

It seems that adding @hv_param is not necessary. for notein. Correct?

**Comment by mhroth** [comment]
@woldhek You are correct. The `@hv_param` markup is only necessary if you want to "expose" the receiver. The receivers will always be there and will always be usable, but you might not want all of them to show up in your plugin interface. In general, you won't need to expose `__hv_notein`.

But you can use Pd's `notein` object to get the midi notes from the DAW. That is working.
